src/chess/Command.java

```diff
@@ -18,17 +18,14 @@
   static Command parse(String str) {
     switch (str.toLowerCase()) {
       case "help":
-        System.out.println("* type 'help' for help");
-        System.out.println("* type 'board' to see the board again");
-        System.out.println("* type 'resign' to resign");
-        System.out.println("* type 'moves' to list all possible moves");
-        System.out.println("* type a square (e.g. b1, e2) to list possible moves for that square");
-        System.out.println("* type a UCI (e.g. b1c3, e7e8q) to make a move");
```

Comment:
I'm not sure what you did, but it seems like you manually mixed your changes and Diego's changes https://github.com/usk83/CICCC-Group-Mini-Project/pull/12

When you're using git, you definitely shouldn't add others changes manually, otherwise, it'll cause conflicts.